ruspy/simulation/simulation.py

```diff
@@ -105,25 +80,4 @@ def simulate(init_dict, ev_known=None, shock=None):
     for _ in range(num_buses):
         period = np.append(period, np.arange(num_periods))
     df["period"] = period.astype(int)
-    return df, unobs, utilities, num_states
-
-
-def get_unobs(shock, num_buses, num_periods):
-    unobs = np.empty(shape=(num_buses, num_periods, 2), dtype=float)
-    shock = (
-        (
-            pd.Series(index=["loc"], data=[-np.euler_gamma], name="gumbel_r"),
-            pd.Series(index=["loc"], data=[-np.euler_gamma], name="gumbel_r"),
-        )
-        if shock is None
-        else shock
-    )
-    dist_func_shocks_maint = getattr(stats, shock[0].name)
-    dist_func_shocks_repl = getattr(stats, shock[1].name)
-    unobs[:, :, 0] = dist_func_shocks_maint.rvs(
-        **shock[0], size=[num_buses, num_periods]
-    )
-    unobs[:, :, 1] = dist_func_shocks_repl.rvs(
-        **shock[1], size=[num_buses, num_periods]
-    )
-    return unobs
+    return df, unobs, utilities
```

Comment:
Can we put the `unobs` just into the simulated data frame? Then the simulated data frame is just a superset of the columns of the observed counterpart.

Reply:
But then I can only see the unobservables of the chosen alternative. This way it allows to explore afterwards decisions for each period, as I have all information on it.

Reply:
Please add two columns.